src/components/matches/questionnaire.component.js

```diff
@@ -61,24 +62,14 @@ var PERSONALITY_TYPES = [
   "Dancing",
 ];*/
 
-const CustomButton = withStyles((theme) => ({
-  root: {
-    color: "#00B183",
-    backgroundColor: "transparent",
-    border: "2px solid #00B183",
-    borderRadius: "8px",
-    font: "inherit",
-    fontSize: "16px",
-    fontWeight: "normal",
-  },
-}))(Button);
-
 export default function Questionnaire(props) {
+  const loginInfo = useSelector((state) => state.Login);
+
   // const [hobbies, setHobbies] = useState({});
   const [successfulPost, setSuccessfulPost] = useState(false);
   const [notLoggedIn, setNotLoggedIn] = useState(props.userId === "");
```

Comment:
Let's comment out line 70 here, looks like we're pulling straight from redux to determine whether or not the user is logged in or not so this isn't being used